rust-engine/src/ast/identifiers/global_id/view.rs

```diff
@@ -800,5 +818,29 @@ mod view_encapsulation {
             Self(inner)
         }
     }
+
+    impl From<FreshModule> for View {
+        fn from(value: FreshModule) -> Self {
+            use crate::ast::diagnostics::{Context, DiagnosticInfo};
+            (DiagnosticInfo {
+            context: Context::NameView,
+            span: Span::dummy(),
+            kind: hax_types::diagnostics::Kind::Unimplemented {
+                issue_id: Some(1779),
+                details: Some(
+                    "Unsupported fresh modules, which should only come from the fstar-phase for bundling"
```

Comment:
Maybe change to "Fresh modules are not implemented yet in the Rust engine" or something in those lines?
See my comment in #1779, it's not so much F* specific